packages/documentation/src/pages/getting-started/common-tasks/updating-formation.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Making Updating Formation and Formataion React 
+tags: formation, formation-react, update, release, publish, npm
+---
+
+## Making updates to formation
+You can find a detailed guide on contributing to formation at [https://dev-design.va.gov/documentation/contributing-to-formation](https://dev-design.va.gov/documentation/contributing-to-formation)
+
+## Making updates to formation-react
+1. Components should be created and tested in vets-website first before being moved over to formation-react
+2. Transfer component to formation-react
+3. Test new node_module component locally. [Check out these instructions on how to do so](/previewing-changes)
```

Comment:
I'm not sure "node_module component" makes sense. Maybe say "Test the formation-react version of the component in vets-website before publishing"